### PR TITLE
chore: fix some bugs

### PR DESCRIPTION
**Pull request by rphmeier**

### CODE REVIEW

**Comment by rphmeier** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/thrumdev/nomt/306?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#306** <a href="https://app.graphite.dev/github/pr/thrumdev/nomt/306?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#305** <a href="https://app.graphite.dev/github/pr/thrumdev/nomt/305?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#304** <a href="https://app.graphite.dev/github/pr/thrumdev/nomt/304?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#303** <a href="https://app.graphite.dev/github/pr/thrumdev/nomt/303?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#302** <a href="https://app.graphite.dev/github/pr/thrumdev/nomt/302?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#301** <a href="https://app.graphite.dev/github/pr/thrumdev/nomt/301?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#300** <a href="https://app.graphite.dev/github/pr/thrumdev/nomt/300?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#299** <a href="https://app.graphite.dev/github/pr/thrumdev/nomt/299?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#296** <a href="https://app.graphite.dev/github/pr/thrumdev/nomt/296?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#295** <a href="https://app.graphite.dev/github/pr/thrumdev/nomt/295?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#293** <a href="https://app.graphite.dev/github/pr/thrumdev/nomt/293?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#292** <a href="https://app.graphite.dev/github/pr/thrumdev/nomt/292?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#297](https://github.com/thrumdev/nomt/pull/297) <a href="https://app.graphite.dev/github/pr/thrumdev/nomt/297?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* **#291** <a href="https://app.graphite.dev/github/pr/thrumdev/nomt/291?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @rphmeier and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: nomt/src/seek.rs [hunk 1]
```diff
@@ -77,6 +80,10 @@ impl SeekRequest {
             panic!("seek past end")
         };
 
+        self.continuations += 1;
+
+        assert!(self.continuations < 10);
```
Comment (gabriele-0201) [inline nomt/src/seek.rs:85]: Why is there this limit? Why is `continue_seek` only callable a maximum of 10 times? Or am I just looking at it wrong?

The trie is made up of a maximum of 42 pages (height), so if continue_seek just walks through one page at a time, isn't 10 too small a limit?

Comment (gabriele-0201) [reply nomt/src/seek.rs:85]: removed in #307

**Review by gabriele-0201 — APPROVED** [review]

**Comment by pepyakin** [comment]
### Merge activity

* **Aug 23, 10:23 AM EDT**: @pepyakin started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/thrumdev/nomt/306).

File: nomt/src/seek.rs (hunk 1 above)
Comment (rphmeier) [reply nomt/src/seek.rs:85]: yeah it's actually a debugging thing I accidentally left in :see_no_evil:

**Pull request merged**